pkg/agent/multicast/mcast_controller.go

```diff
@@ -60,9 +61,10 @@ type mcastGroupEvent struct {
 type GroupMemberStatus struct {
 	group net.IP
 	// localMembers is a set for the local Pod's interface name which has joined in the multicast group.
```

Comment:
Please update this comment. `localMembers` is not a set

Reply:
updated.